src/components/ItemSelection.tsx

```diff
@@ -118,9 +118,8 @@ const ItemSelection: React.FC<{
           userSelection={userSelection}
         />
       ) : null}
-      {console.log(allItems)}
-      {console.log(itemStack)}
-      {userSelection.userItems.length === 15 ? (
+
+      {userSelection.userItems.length === 15 ? (        
```

Comment:
would recommend cleaning up the conditional rendering logic of this whole `return` to the one we discussed 

`userSelection.userItems.length < 15 ? <ItemBar/><ItemSelection/> : <StackCalcDisplay/>`